<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GraphGenerator</class>
 <widget class="QMainWindow" name="GraphGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>1056</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GraphGenerator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addNodeBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add node</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addEdgeBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add edge</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateArrowsBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Update arrows</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="printGraph">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>1180</width>
      <height>920</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteNodeBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete node</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteEdgeBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete edge</string>
    </property>
   </widget>
   <widget class="QPushButton" name="floydBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>760</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Floyd's algorithm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dijkstraBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dijkstra's algorithm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bfsBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Breadth-first search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dfsBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Depth-First Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tspBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>850</y>
      <width>200</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Travelling Salesman
Problem</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>934</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
